launch_ros/package.xml

```diff
@@ -6,8 +6,9 @@
   <name>launch_ros</name>
   <version>0.15.0</version>
   <description>ROS specific extensions to the launch tool.</description>
+  <maintainer email="aditya.pande@osrfoundation.org">Aditya Pande</maintainer>
   <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
-  <maintainer email="michael.jeronimo@openrobotics.org">Michael Jeronimo</maintainer>
```

Comment:
Historically, we've added `<author>` tags for previous maintainers to recognize their contribution.

Reply:
Good point. See https://github.com/ros2/launch_ros/pull/284/commits/3f5ab81c15cbea1722a54d22bebe9bfb963a8cdd.